app/assets/stylesheets/reskin/moneyhelper_styling_common.scss

```diff
@@ -0,0 +1,210 @@
+// -- Headers ----------------------------------------------------------------------
+h1,h2,h3,h4,h5,h6{
+    font-family: 'Roobert-Bold', 'Times';
```

Comment:
This font-family declaration is inconsistent with other uses of the rule elsewhere and also with itself: we would not expect a sans-serif and a serif font to be listed in the same font-family.